docs/0_prerequisites.md

```diff
@@ -1,10 +1,15 @@
 # 💼 Prerequisites
 
-In order to use ReVanced Manager, certain requirements must be met.
+To use ReVanced Manager, certain requirements have to be met.
 
 ## 🤝 Requirements
 
-- An Android device running Android 8 or higher
+- Android device running Android 8.0 or higher
+- System that support running at least `arm64-v8a`, `armeabi-v7a`, `x86_64`, or `x86` application
```

Comment:
Pretty sure every commercial android device runs one of these.

Reply:
Fair enough since MIPS64 and RISC-V are unsupported by Android nowadays.

```suggestion
```